<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageLoader_Dir</class>
 <widget class="QDockWidget" name="ImageLoader_Dir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageLoader_Dir</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_Dir">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
        <item>
         <property name="text">
          <string>*</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_Add">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_Reset">
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>文件夹</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Run">
      <property name="text">
       <string>运行</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
